android/src/main/kotlin/com/jhomlala/better_player/BetterPlayer.kt

```diff
@@ -648,8 +655,26 @@ internal class BetterPlayer(
             })
             mediaSession.isActive = true
             mediaSessionConnector = MediaSessionConnector(mediaSession)
-            mediaSessionConnector?.setPlayer(exoPlayer)
-
+            mediaSessionConnector?.apply {
+                setPlayer(exoPlayer)
+                if (Build.MANUFACTURER.lowercase() == "samsung" && Build.VERSION.SDK_INT == Build.VERSION_CODES.R) {
+                    // LIVE
```

Comment:
Question:
Do we need to implement VOD or LIVE separately?
I'm curious since there is no flag at here and I think these lines will be executed on both LIVE and VOD case.

Request:
If you have, can you write any link about this SAMSUNG device issue as a comment too?

Reply:
@HiroshiOshiro 
- Implementing these two parts is necessary because if only one is implemented, the other will not function correctly ( not show the title in the notification)
- I only have stackoverflow question link, I can write it if you want. There are no official documents regarding this issue.

Reply:
@mvn-datnguyen-hn 
Please write "stackoverflow question link" into PR description 🙏

Reply:
> Implementing these two parts is necessary because if only one is implemented, the other will not function correctly ( not show the title in the notification)

I see, thanks.